posts/zzz_DO_NOT_EDIT_admiral_1.3.../admiral_1.3.0:_documentation_go!!.qmd

```diff
@@ -0,0 +1,152 @@
+---
+title: "admiral 1.3.0: documentation go!!"
+author:
+  - name: Ben Straub 
+  - name: Edoardo Mancini
+description: "{admiral} 1.3 is here - say hello to supercharged examples, bug fixes and more!"
+# Note that the date below will be auto-updated when the post is merged.
+date: "2025-06-20"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [ADaM]
+# Feel free to change the image
+image: "admiral.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_admiral_1.3..."
+library(link)
+link::auto()
+```
+
+<!--------------- post begins here ----------------->
+
+![](admiral.png){fig-align="center" width="220"}
+
+The second half of 2025 brings us once again to an exciting {admiral} release - doesn't time fly?! Read through this blog post to learn more what to expect in {admiral} 1.3.
+
+## Headline update: documentation that gets you
+
+Our crowning update this release cycle is...
+
+**Documentation updates!**
+
+Hmmm, doesn't sound fancy enough? What about...
+
+<span style="font-size: 2em;
+              background: linear-gradient(to right, red, orange, yellow, green, blue, indigo, violet);
+              -webkit-background-clip: text;
+              -webkit-text-fill-color: transparent;">
+Documentation updates!!
+</span>
+
+Still not that interesting? Well, let us change your mind with an example. Here's what the examples section of the reference page for admiral::derive_extreme_records() looked like in {admiral} 1.2:
```

Comment:
```suggestion
Here's what the examples section of the reference page for admiral::derive_extreme_records() looked like in {admiral} 1.2:
```

Reply:
Gonna reject this one as discussed to keep a bit of pizazz